Makefile

```diff
@@ -0,0 +1,13 @@
+PREFIX = /usr
+
+all:
+	@echo Run \'make install\' to install notflix.
+
+install:
+	@mkdir -p $(DESTDIR)$(PREFIX)/bin
+	@cp -p notflix $(DESTDIR)$(PREFIX)/bin/notflix
+	@chmod 755 $(DESTDIR)$(PREFIX)/bin/notflix
+
+uninstall:
+	@rm -rf $(DESTDIR)$(PREFIX)/bin/notflix
```

Comment:
Why use recursive flag if the target is not a directory? I would go with:

```make
uninstall:
	@rm -f $(DESTDIR)$(PREFIX)/bin/notflix
```

Reply:
Oh whoops. Force of habit. My bad 😅